R/any_is_na_linter.R

```diff
@@ -1,8 +1,8 @@
-#' Require usage of anyNA over any(is.na(.))
+#' Require usage of `anyNA(x)` over `any(is.na(x))`
 #'
-#' [anyNA()] exists as a replacement for `any(is.na(.))` which is more efficient
-#'   for simple objects, and in the worst case is the same efficiency. Therefore
-#'   it should be used in all situations instead of the latter.
+#' [anyNA()] exists as a replacement for `any(is.na(x))` which is more efficient
+#'   for simple objects, and has the same efficiency in the worst case scenario.
```

Comment:
parallel phrasing with `any_duplicated_linter`: "is at worst equally efficient"

Reply:
Done.